.github/workflows/compile-examples.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           - arduino:samd:nano_33_iot
           - arduino:megaavr:uno2018:mode=on
           - arduino:mbed:nano33ble
```

Comment:
should we also change this to `arduino:mbed_nano:nano33ble` ?

Reply:
Definitely, and I was itching to make that change, but figured I should stick to the exact request I received, which was to add the Nano RP2040 Connect to the CI.

I'm happy to throw that change in this PR if you like though.

Reply:
Oh, I see. Don't worry, it's perfect to submit that change in a new PR.